utility_scripts/deploy/deploy_hyperion.py

```diff
@@ -71,11 +75,45 @@ def get_hyperion_release_dir_from_args() -> str:
     args = parser.parse_args()
     if args.beamline == "dev":
         print("Running as dev")
-        return "/scratch/30day_tmp/hyperion_release_test/bluesky"
+        return DEV_DEPLOY_LOCATION
     else:
         return f"/dls_sw/{args.beamline}/software/bluesky"
 
 
+def create_environment_from_control_machine():
+    ssh_client = paramiko.SSHClient()
+
+    # This adds i03-control to your client's known hosts.
+    ssh_client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
```

Comment:
is it really necessary to have username/password for this? I don't have to enter them when I `ssh i03-control` normally so I would prefer an automated solution, probably a little more secure too

Reply:
I tested out just the SSH part, and for me I do need the credentials. Potentially if you set up SSH properly in your `bashrc` then this won't be needed, but then this needs to be a prerequisite for deployment

Reply:
@DominicOram  @rtuck99 do you have opinions on what is preferred?

Reply:
I think so, it might look a little more clunky, but I guess it would be better if it worked it out for us

Reply:
Yh, lets try it. This is always going to be a bit temporary anyway